### DISCUSSION TITLE
Request for the retention of the old Horus Heresy rules

### DISCUSSION THREAD

**Discussion by marius-karus** [BSData/horus-heresy#2064] [Q&A] [answered]
Hello there.
I hope that this is the right place to ask, but I wanted to make a small request regarding the rule-set for our beloved Horus Heresy. Some people, myself included, would like to continue playing the old edition of Horus Heresy, regardless of the release of HH 2.0. Could you please leave the old Horus Heresy battlescribe rules and Mournival rules, so that we can use them?  By all means, do the rule-set for 2.0, but please leave the old rules, as well.

**Comment by Mayegelt** [comment, marked as answer]
Hello. The plan is that this repo remain open, and possibly active (depending on if the community wish to continue to support it with things like the fanmade content.
I started some very light work on a new repo for Horus Heresy 2022, but left it private for now, while I work out the core of what is still required.

**Reply by nstephenh** [reply]
Just an update, we had the rules for both games in the same repo for a while, before I eventually moved the 1.0 rules here:
https://github.com/BSData/horus-heresy-1e

We're not actively working on them but if anyone does want to they are free to.
